bucket_api.go

```diff
@@ -57,6 +57,10 @@ func (bucket *Bucket) Close(_ context.Context) {
 	bucket.mutex.Lock()
 	defer bucket.mutex.Unlock()
 
+	bucket.closed = true
+}
+
+func (bucket *Bucket) _closeAllInstances() {
```

Comment:
What are the 'AllInstances' here, since it looks like it's just closing this single bucket's connection?

Reply:
Renamed this `closeSQliteDB` although it really closes more than that. If this function is called on any bucket, it will shut down all Bucket that have a matching name. Any suggestion of a better name to reflect would be welcome.